modules/quota-establishment-api.adoc

```diff
@@ -141,7 +144,7 @@ $ podman tag docker.io/library/nginx example-registry-quay-quay-enterprise.apps.
 $ podman push --tls-verify=false example-registry-quay-quay-enterprise.apps.docs.gcp.quaydev.org/testorg/nginx
 ----
 
-. To view the quota report for the repositories in the organization, use the */api/v1/repository* endpoint:
+. Use the */api/v1/repository* endpoint to view the quota report for the repositories in the organization:
```

Comment:
In a recent OpenShift review for odo, it was suggested to use this format for instructions...